src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/Resources/Strings.resx

```diff
@@ -214,7 +214,7 @@
     <value>All methods must be declared in the same partial definition of a 'GeneratedComInterface'-attributed interface type to ensure reliable calculation for virtual method table offsets.</value>
   </data>
   <data name="MethodNotDeclaredInAttributedInterfaceMessage" xml:space="preserve">
-    <value>The method '{0}' is declared on a different partial definition of the interface '{1}' than the definition that has the 'GeneratedComInterface' attribute</value>
+    <value>The method '{0}' is declared on a different partial definition of the interface '{1}' than the definition that has the 'GeneratedComInterface' attribute. Interop code for '{0}' will not be generated</value>
```

Comment:
If this is always going to be an error that breaks the build I don't think we need the extra part. I think we should leave the diagnostic as is or make the `MethodNotDeclaredInAttributedInterface` diagnostic a warning.